capstone_project/src/main/java/com/google/sps/data/Resource.java

```diff
@@ -17,57 +17,45 @@
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
 import java.util.HashMap;
 import java.util.Map;
 
 /** Associates a category, from the Natural Language API taxonomy, to a link to a resource */
 public class Resource {
-  private final DatastoreService datastore;
-  private final String category;
-  private final String resource;
+  private static final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
I don't think a DatastoreService should be a static class member. Just have DatastoreService be a parameter for storeResource and addPreexistingResources.